### ISSUE TITLE
mod-text does not include image alt when displaying image

### ISSUE THREAD

**Issue by michaelclaesson** [helsingborg-stad/Modularity#792]
When you insert an image into a mod-text module this image gets converted to a proper image component with figure. This however does not have the alt text tied to the image in the media library and source code.

### Editor with alt text
<img width="1227" alt="Image" src="https://github.com/user-attachments/assets/87e31e43-af16-490c-a452-1407b0096217" />

### Displayed module
<img width="405" alt="Image" src="https://github.com/user-attachments/assets/3c86dd31-9827-417a-86fe-8d4ceccc4fc3" />

### Source elements without alt text
<img width="1123" alt="Image" src="https://github.com/user-attachments/assets/dc106e1e-00af-4f0d-b047-3fe069570cb9" />
